tests/test_oneclick.py

```diff
@@ -394,4 +396,84 @@ def test_release_hold(self):
         eq_(True, success)
 
 
+    def test_update_licensepool_for_identifier(self):
+        """Test the OneClick implementation of the update_availability method
+        defined by the CirculationAPI interface.
+        """
+
+        # Create a LicensePool that needs updating.
+        edition, pool = self._edition(
+            identifier_type=Identifier.ONECLICK_ID,
+            data_source_name=DataSource.ONECLICK,
+            with_license_pool=True
+        )
+
+        # We have never checked the circulation information for this
+        # LicensePool. Put some random junk in the pool to verify
+        # that it gets changed.
+        pool.licenses_owned = 10
+        pool.licenses_available = 5
+        pool.patrons_in_hold_queue = 3
+        eq_(None, pool.last_checked)
+
+        isbn = pool.identifier.identifier.encode("ascii")
+
+        self.api.update_licensepool_for_identifier(isbn, False)
+
+        # The availability information has been updated, as has the
+        # date the availability information was last checked.
+        eq_(10, pool.licenses_owned)
+        eq_(0, pool.licenses_available)
+        eq_(3, pool.patrons_in_hold_queue)
+        assert pool.last_checked is not None
+
+        self.api.update_licensepool_for_identifier(isbn, True)
+        eq_(999, pool.licenses_available)
+
+
+
+class TestCirculationMonitor(OneClickAPITest):
+
+    def test_process_availability(self):
+        with temp_config() as config:
+            config[Configuration.INTEGRATIONS]['OneClick'] = {
+                'library_id' : 'library_id_123',
+                'username' : 'username_123',
+                'password' : 'password_123',
+                'remote_stage' : 'qa', 
+                'base_url' : 'www.oneclickapi.test', 
+                'basic_token' : 'abcdef123hijklm', 
+                "ebook_loan_length" : '21', 
+                "eaudio_loan_length" : '21'
+            }
+            monitor = OneClickCirculationMonitor(self._db)
+            monitor.api = MockOneClickAPI.from_config(self._db)
+
```

Comment:
I'm not sure I fully grasp the temp_config usage.  Is it that the config files aren't being read during unit testing?  I'll ask in person.